### DISCUSSION TITLE
form 验证通过无法取消错误提示

### DISCUSSION THREAD

**Discussion by pengyinghao** [TaleLin/lin-ui#1273] [求助]
form-item  picker配合input 禁用状态下    如何验证是否填写了内容呢
![image](https://user-images.githubusercontent.com/34115313/104923370-eed8ef80-59d6-11eb-9062-f3db142cd2af.png)
![image](https://user-images.githubusercontent.com/34115313/104923509-1af47080-59d7-11eb-8b04-01ece0693d52.png)

**Comment by juzi214032** [comment]
你目前这个验证有什么问题吗
